lagom-eventstore-example/README.md

```diff
@@ -38,8 +38,8 @@ Follow these steps to get a local copy of this project and configure it to conne
     ```
 3.  To supply the configuration, do one of the following:
```

Comment:
I think "one of the following" might be another copy/paste that doesn't apply here.

Reply:
Fixed in 72aa123